projects/Inventaire/default.nix

```
@@ -27,7 +27,7 @@
         };
         settings-defaults = {
           text = "Default settings";
-          url = "https://codeberg.org/inventaire/inventaire/src/branch/main/config/default.cjs";
+          url = "https://codeberg.org/inventaire/inventaire/src/branch/main/config/default.ts";
```

Perhaps a permalink would be better?


```suggestion
          url = "https://codeberg.org/inventaire/inventaire/src/commit/fd2083a2e19c690c198351d81ed34a8857e0ead6/config/default.ts";
```